documentation/Kubecheck.md

```diff
@@ -317,6 +318,15 @@ This test checks if defined package repositories are available from nodes.
 
 This test checks if needed package are available from nodes.
 
+###### 015 Kernel version
+
+*Task*: `software.kernel.version`
+
+This test checks the Linux kernel version for a `bad kernel version`, if it is equal to a bad one, it issues a warning
```

Comment:
Lets use more freindly term 'unstable kernel version'. Change here and in the rest of places in the code and User message

Reply:
fixed problem